web/directory/static-compiled/admin/css/base.css

```diff
@@ -0,0 +1,978 @@
+/*
```

Comment:
Not a blocker, but I have to wonder if it is necessary to have the stuff in web/directory/static-compiled checked into the repo?  I far rather see this come from somewhere else.  128 files is pretty big for a "boilerplate".

Reply:
Honestly I'm surprised this is not git-ignored by default in impact-api. But I didn't want to spend time figuring it out, so I just included it. Generally speaking, I expect that if @shanbady goes over this Repo he'll be able to make quick clean up gains.

Reply:
I agree that generated/compiled/built files should not be committed to the repo. I added a note to this effect on the follow-on ticket AC-5091, just to remind us.